[feat] Improve the gs command tool to support for Alibaba Cloud Linux

**Is your feature request related to a problem? Please describe.**
When I use the gs shell command to install the deps for  in the Alibaba Cloud Linux, I get the error as follows.

```shell
./gs install-deps dev
Setting the environment for development.

[WARNING] GRAPHSCOPE_HOME will set to source root (/root/GraphScope) for development.
[WARNING] To use you assigned GRAPHSCOPE_HOME, export GRAPHSCOPE_ENV=prod.

Read the env: GRAPHSCOPE_HOME=/root/GraphScope
args:
- ${args[--from-local]} = /tmp/gs-local-deps
- ${args[--install-prefix]} = /opt/graphscope
- ${args[type]} = dev
- ${args[--v6d-version]} = main
[WARNING] Please note that I am running as root.
Installing dev dependencies for GraphScope on Alibaba Cloud Linux-3...
[ERROR] The script is only support platforms of Ubuntu/CentOS/macOS
```

**Describe the solution you'd like**

Support to install the deps for `gae, gie, gle` on the Alibaba Cloud Linux.

--- a/gs
+++ b/gs
@@ -523,7 +523,15 @@ cyan_underlined() { print_in_color "\e[4;36m" "$*"; }
 
 # src/lib/get_os_version.sh
 get_os_version() {
-  if [ -f /etc/os-release ]; then
+  if [ -f /etc/centos-release ]; then
+    # Older Red Hat, CentOS, Alibaba Cloud Linux etc.
+    PLATFORM=CentOS
+    OS_VERSION=$(sed 's/.* \([0-9]\).*/\1/' < /etc/centos-release)
+    if grep -q "Alibaba Cloud Linux" /etc/centos-release; then
+      PLATFORM="Aliyun_based_on_CentOS"
+      OS_VERSION=$(rpm -E %{rhel})
+    fi
+  elif [ -f /etc/os-release ]; then
     # freedesktop.org and systemd
     . /etc/os-release
     PLATFORM="${NAME}"
@@ -541,10 +549,6 @@ get_os_version() {
     # Older Debian/Ubuntu/etc.
     PLATFORM=Debian
     OS_VERSION=$(cat /etc/debian_version)
-  elif [ -f /etc/centos-release ]; then
-    # Older Red Hat, CentOS, etc.
-    PLATFORM=CentOS
-    OS_VERSION=$(sed 's/.* \([0-9]\).*/\1/' < /etc/centos-release)
   else
     # Fall back to uname, e.g. "Linux <version>", also works for BSD, Darwin, etc.
     PLATFORM=$(uname -s)
@@ -1030,7 +1034,7 @@ install_vineyard() {
         -DCMAKE_INSTALL_PREFIX="${V6D_PREFIX}" \
         -DBUILD_VINEYARD_TESTS=OFF \
         -DBUILD_SHARED_LIBS=ON \
-        -DBUILD_VINEYARD_PYTHON_BINDINGS=ON \
+        -DBUILD_VINEYARD_PYTHON_BINDINGS=ON  \
         -DBUILD_VINEYARD_GRAPH_WITH_GAR=ON
   make -j"${jobs}"
   make install
@@ -1490,8 +1494,8 @@ gs_install_deps_command() {
   fi
 
   check_os_compatibility() {
-    if [[ "${OS_PLATFORM}" != *"Ubuntu"* && "${OS_PLATFORM}" != *"CentOS"* && "${OS_PLATFORM}" != *"Darwin"* ]]; then
-      err "The script is only support platforms of Ubuntu/CentOS/macOS"
+    if [[ "${OS_PLATFORM}" != *"Ubuntu"* && "${OS_PLATFORM}" != *"CentOS"* && "${OS_PLATFORM}" != *"Darwin"* && "${OS_PLATFORM}" != *"Aliyun"* ]];  then
+      err "The script is only support platforms of Ubuntu/CentOS/macOS/AliyunOS"
       exit 1
     fi
 
@@ -1641,17 +1645,22 @@ gs_install_deps_command() {
     if [[ "${OS_PLATFORM}" == *"Ubuntu"* ]]; then
       ${SUDO} apt-get update -y
       DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC ${SUDO} apt-get install -y ${BASIC_PACKAGES_UBUNTU[*]}
-    elif [[ "${OS_PLATFORM}" == *"CentOS"* ]]; then
+    elif [[ "${OS_PLATFORM}" == *"CentOS"* || "${OS_PLATFORM}" == *"Aliyun"* ]]; then
       if [[ "${OS_VERSION}" -eq "7" ]]; then
         ${SUDO} yum install -y ${BASIC_PACKAGES_CENTOS_7[*]}
         ${SUDO} yum install -y ${ADDITIONAL_PACKAGES_CENTOS_7[*]}
       else
-        sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
-        sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
-        ${SUDO} yum install -y 'dnf-command(config-manager)'
-        ${SUDO} dnf install -y epel-release
+        if [[ "${OS_PLATFORM}" == *"Aliyun"* ]]; then 
+          ${SUDO} yum install -y 'dnf-command(config-manager)'
+          ${SUDO} dnf install -y epel-release --allowerasing
+        else
+          sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+          sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
+          ${SUDO} yum install -y 'dnf-command(config-manager)'
+          ${SUDO} dnf install -y epel-release
+          ${SUDO} dnf config-manager --set-enabled powertools
+        fi
         ${SUDO} dnf config-manager --set-enabled epel
-        ${SUDO} dnf config-manager --set-enabled powertools
         ${SUDO} yum install -y ${BASIC_PACKAGES_CENTOS_8[*]}
         ${SUDO} yum install -y ${ADDITIONAL_PACKAGES_CENTOS_8[*]}
       fi

